brainclient/brainClient.go

```diff
@@ -0,0 +1,87 @@
+package brainclient
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/bits-and-blooms/bloom/v3"
+)
+
+type BrainClient interface {
+	CheckUrl(url string) (bool, error)
+	DownloadBloomFilter() *bloom.BloomFilter
+}
+
+type Client struct {
+	brainAddress       string
+	indicatorsEndpoint string
+	detailsEndpoint    string
+}
+
+func NewClient(brainAddr string, indicatorsEndpoint string, detailsEndpoint string) *Client {
```

Comment:
I'm just thinking, not sure about it: maybe we can return *BrainClient instead of *Client? In java i might return the interface instead that the concrete type. But it depends why there is here an interface (so maybe both ways can be ok).